src/axom/sina/core/Document.cpp

```diff
@@ -43,8 +49,42 @@ namespace
 char const RECORDS_KEY[] = "records";
 char const RELATIONSHIPS_KEY[] = "relationships";
 char const SAVE_TMP_FILE_EXTENSION[] = ".sina.tmp";
+// These two are used in switch statements--could be cleaner
+enum AppendFields
+{
+  ID_FIELD,
+  LOCAL_ID_FIELD,
+  TYPE_FIELD,
+  APPLICATION_FIELD,
+  USER_FIELD,
+  VERSION_FIELD,
+  DATA_FIELD,
+  USER_DEFINED_FIELD,
+  FILES_FIELD,
+  CURVE_SETS_FIELD,
+  LIBRARY_DATA_FIELD,
+  UNKNOWN_FIELD
+};
+// Really we should have one single source of truth on this one--toplevel Sina, perhaps?
+static const std::map<std::string, AppendFields> appendFieldStrings {
+  {"id", AppendFields::ID_FIELD},
+  {"local_id", AppendFields::LOCAL_ID_FIELD},
+  {"type", AppendFields::TYPE_FIELD},
+  {"application", AppendFields::APPLICATION_FIELD},
+  {"user", AppendFields::USER_FIELD},
+  {"version", AppendFields::VERSION_FIELD},
+  {"data", AppendFields::DATA_FIELD},
+  {"user_defined", AppendFields::USER_DEFINED_FIELD},
+  {"files", AppendFields::FILES_FIELD},
+  {"curve_sets", AppendFields::CURVE_SETS_FIELD},
+  {"library_data", AppendFields::LIBRARY_DATA_FIELD},
+};
 }  // namespace
 
+std::vector<std::string> const CURVE_CATEGORIES = {"dependent", "independent"};
+#define UNUSED(x) \
+  (void)(x)  // TODO: axom likely has a better preference on how to suppress the warning
```

Comment:
Please remove this `UNUSED` macro, and replace all uses with `AXOM_UNUSED_VAR`  (already included via `axom/core.hpp`)